app/javascript/redux/apis/interviews.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export default axios.create({
+  baseURL: "http://localhost:3000",
+});
```

Comment:
Try not to use axios in the future. ES6 fetch function is enough for API fetching.